### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by nasbench**
### Summary of the Pull Request

This PR updates the README.md file. You can check how the new update look [here](https://github.com/nasbench/sigma/blob/update-readme/README.md)

image

### Changelog

chore: update main README.md file

### Example Log Event

N/A

### Fixed Issues

N/A

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,20 +14,20 @@ If you find a false positive or would like to propose a new detection rule idea
...
 
 3. Make your changes and commit them to your branch:
 
-```
+```bash
 git add .
 git commit -m "Your commit message"
 ```
 
 4. Push your changes to your fork:
```
Comment (frack113) [inline CONTRIBUTING.md:28]: 3 bis is to test with 
```bash
python tests/test_logsource.py
python tests/test_rules.py
```

File: README.md [hunk 2]
```diff
@@ -1,156 +1,103 @@
...
 
-# Specification
+## 🏗️ Rule Creation
 
-The specifications can be found in the [Sigma-Specification](https://github.com/SigmaHQ/sigma-specification) repository.
+To start writing Sigma rules please check the following guides:
 
-The current specification is a proposal. Feedback is requested.
+* [How to Write Sigma Rules - Nextron Systems](https://www.nextron-systems.com/2018/02/10/write-sigma-rules/)
+* [Rule Creation Guide](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide)
```
Comment (frack113) [inline README.md:46]: I prefer SigmaHQ links first
* [SigmaHQ Rule Creation Guide](https://github.com/SigmaHQ/sigma/wiki/Rule-Creation-Guide)
* [How to Write Sigma Rules - Nextron Systems](https://www.nextron-systems.com/2018/02/10/write-sigma-rules/)

File: README.md [hunk 3]
```diff
@@ -1,156 +1,103 @@
...
-# Sigma
+<a href="https://opensourcesecurityindex.io/" target="_blank" rel="noopener">
+<img style="width: 282px; height: 56px" src="https://opensourcesecurityindex.io/badge.svg" alt="Open Source Security Index - Fastest Growing Open Source Security Projects" width="282" height="56" />
+</a>
 
-Generic Signature Format for SIEM Systems
+## Explore Sigma
 
-# What is Sigma
+To start exploring the Sigma ecosystem, please visit the official website [sigmahq.io](https://sigmahq.io)
```
Comment (frack113) [inline README.md:17]: The information that it is a rule deposit is missing.
a "Welcome to the SigmaHQ rule repository" ?

Comment (nasbench) [reply README.md:25]: Added something that solve this. Take a look again.

**Review by frack113 — APPROVED** [review]

**Review by Neo23x0 — APPROVED** [review]

**Review by thomaspatzke — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1,156 +1,111 @@
-[![sigma build status](https://github.com/SigmaHQ/sigma/actions/workflows/sigma-test.yml/badge.svg?branch=master)](https://github.com/SigmaHQ/sigma/actions?query=branch%3Amaster)
+# Sigma - Generic Signature Format for SIEM Systems
```
Comment (sifex) [inline README.md:1]: I had a tweaked header design if you want, imo I like centered better
```
<a href="https://sigmahq.io/">
<p align="center">
<br />
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="./images/sigma_logo_dark.png">
  <img height="135" alt="Sigma Logo" src="./images/sigma_logo_light.png">
</picture>
</p>
</a>
<br />

<p align="center">
<a href="https://github.com/SigmaHQ/sigma/actions?query=branch%3Amaster"><img src="https://github.com/SigmaHQ/sigma/actions/workflows/sigma-test.yml/badge.svg?branch=master" alt="Sigma Build Status"></a> <a href="https://sigmahq.io/"><img src="https://cdn.jsdelivr.net/gh/SigmaHQ/sigmahq.github.io@master/images/Sigma%20Official%20Badge.svg" alt="Sigma Official Badge"></a> <img alt="GitHub Repo stars" src="https://img.shields.io/github/stars/SigmaHQ/sigma">
<img alt="GitHub all releases" src="https://img.shields.io/github/downloads/SigmaHq/Sigma/total">
<br />
<a href="https://opensourcesecurityindex.io/" target="_blank" rel="noopener">
<img style="width: 170px;" src="https://opensourcesecurityindex.io/badge.svg" alt="Open Source Security Index - Fastest Growing Open Source Security Projects" width="170" />
</a>
</p>

## About Sigma
```

Comment (nasbench) [reply README.md:1]: Thanks, I added your suggestion.

**Pull request merged**